specification/postgresql/resource-manager/readme.md

```diff
@@ -90,6 +100,7 @@ input-file:
 - Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json
```

Comment:
Will there be a new file (postgresql.json) in resource-manager path like below ? 
F:\source\azure-rest-api-specs\specification\postgresql\resource-manager\Microsoft.DBforPostgreSQL\stable\2017-12-01\postgresql.json